### ISSUE TITLE
microsoft listed as an individual, not an organisation

### ISSUE THREAD

**Issue by richlv** [mapillary/mapillary_issues#2713]
minor issue, but microsoft is listed as an individual, not an organisation :)

**Comment by peterneubauer** [comment]
Indeed, something to fix. https://www.mapillary.com/app/stats/85633793?user=microsoft

_mapillary_issues_ is no longer an active repository.

Let’s continue discussing features in our forum! We will import this issue as topic to the [Mapillary Forum](http://forum.mapillary.com) in the coming weeks. We’ll notify here once it’s been done. See you there!
